tests/beaker_tests/cisco_interface/routedintf_provider_negatives.rb

```diff
@@ -526,6 +526,174 @@
     logger.info("Check interface instance absence on agent :: #{result}")
   end
 
+  # @step [Step] Requests manifest from the master server to the agent.
```

Comment:
Did you mean to add these tests to routedintf_provider_negatives.rb.  The tests are broken down into 3 categories (default, non-default, negatives) 

default = set all manifest values to default and test
non_default = set all manifest values to non-default and test
negative = set manifest values to invalid input and test.